ci/blank.yml

```
@@ -2,14 +2,17 @@
 
 name: CI
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the $default-branch branch
+# Controls when the action will run. 
 on:
+  # Triggers the workflow on push or pull request events but only for the $default-branch branch
```

```suggestion
  # Triggers the workflow on push or pull request events but only for the default branch
```

Maybe? Reads a little easier.

---

@mscoutermarsh `$default-branch` is automatically replaced with the name of the actual default branch when you use this workflow

---

Yeah I know, I read it in my head as "default branch default branch"